### PR TITLE
Added transfer project

### PR DESCRIPTION
**Pull request by vkuznet**
Please take new CMS project for Gsoc

### CODE REVIEW

File: _gsocproposals/proposal_TRANSFER.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Next generation of CMS data replication system
+layout: plain
+project: TRANSFER
+organization: Cornell, USA and Univ. of Bologna, Italy
```
Comment (hegner) [inline _gsocproposals/proposal_TRANSFER.md]: can you change that to CMS? And I will add CMS to the participating organizations on http://hepsoftwarefoundation.org/activities/gsoc.html

**Comment by jouvin** [comment]
I also suggest that we merge #76 first to avoid the problem we had with #77.

File: _gsocproposals/proposal_TRANSFER.md (hunk 1 above)
Comment (vkuznet) [inline _gsocproposals/proposal_TRANSFER.md]: @hegner, done, I put CMS as organization, the univ were requested by Sergei though.

**Comment by hegner** [comment]
Has been handled with #84

**Pull request closed without merging**